components/dash-core-components/src/components/RangeSlider.react.js

```diff
@@ -125,6 +125,29 @@ RangeSlider.propTypes = {
             'bottomLeft',
             'bottomRight',
         ]),
+        /**
+         * Format to apply to the tooltip
+         * The string must contain `{value}`
+         */
+        format: PropTypes.string,
```

Comment:
I don't know about `format` - based on that I'd expect you to be able to choose the precision, whether or not to use exponential notation, etc, but it's really only about prefixing and suffixing the default string representation (or the transformed representation) of the number. So I might change it to `template` instead, and say something like "Template string to display the tooltip in. Must contain `{value}`, which will be replaced with either the default string representation of the value or the result of the `transform` function if there is one"